jupyterhub/values.yaml

```diff
@@ -148,15 +148,22 @@ proxy:
         memory: 512Mi
   nginx:
     image:
-      name: quay.io/kubernetes-ingress-controller/nginx-ingress-controller
-      tag: 0.15.0
+      name: nginx
+      tag: 1.17-alpine
     proxyBodySize: 64m
     hstsIncludeSubdomains: 'false'
     resources: {}
-  lego:
+  traefik:
     image:
-      name: jetstack/kube-lego
-      tag: 0.1.7
+      name: traefik
+      tag: v2.1
+    proxyBodySize: 64m
```

Comment:
Do we need this for the certificate issuer? Or does normal traffic flow through this proxy as well?

Reply:
I need to figure out if this is needed. And yes, normal traffic flows through this proxy - it replaces the nginx ingress from earlier.